app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  attr_writer :current_user
+
+  def current_user
+    user_id = session[:user_id]
```

Comment:
Here instead of `session[:user_id]`, please, use `cookies.permanent`

Reply:
implemented